pipelines/docker-build-multi-platform-oci-ta/docker-build-multi-platform-oci-ta.yaml

```diff
@@ -256,7 +256,12 @@ spec:
       operator: in
       values:
       - "false"
-  - name: clair-scan
+  - matrix:
+      params:
+      - name: image-platform
```

Comment:
Why does every task use a different name for the platform param?

preflight uses `platform`, clamav uses `image-arch` and now clair uses `image-platform`

Reply:
Good question.

Originally, I used the name `image-arch` for clamav-scan, but in Clair I realized it might not be the best choice since the value is actually a platform (e.g., linux/amd64). In any case, the parameter name should be consistent across tasks. I can create a PR later to update them all to use the same name `image-platform`.

Reply:
Yes, that would be nice :pray: